### PR TITLE
Development contract

### PR DESCRIPTION
**Pull request by NickGonzalez04**
### What changes were made?

- `Contract Documentation` 
- `Updated Contract Code`
- `ReadMe Scaffolding for Contract & Development Overview`
- `Created separate deploy script files for local and Rinkeby Testnet deployments`

### Is there any background info about the change?

- Just a formatted update for current state Twali Contracts and in the near term iterations to build off of.
- Contract documentation for the first version the contract and describing in detail the contract state, status, and functional actions.

### Are there any state changes? Mention key ones (if any)

- State changes should be the same. If there is any new changes it will naming conventions to match client side and database.

### Screenshots or Gifs✨

Screen Shot 2022-05-24 at 4 26 37 PM
Screen Shot 2022-05-25 at 10 15 47 AM
Screen Shot 2022-05-25 at 10 15 33 AM

### Any new dependencies? Why were they added?
N/A

### Shortcut Link

`Contract Deployment` - https://app.shortcut.com/twali/story/886/deploy-smart-contracts-to-ethereum-mainnet
`Contract Documentation` - https://app.shortcut.com/twali/story/893/twali-contract-v1-documentation

### CODE REVIEW

File: docs/TwaliContract_v1.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Twali Contract Documention
+
```
Comment (nagmak) [inline docs/TwaliContract_v1.md:2]: documentation on POINT fam 🔥

File: hardhat.config.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+require("@nomiclabs/hardhat-waffle");
+require("@nomiclabs/hardhat-etherscan");
+require("hardhat-gas-reporter");
+require("dotenv").config();
+
+console.log(process.env.ADDRESS_KEY);
```
Comment (nagmak) [inline hardhat.config.js:6]: Is this okay to console log here?

File: hardhat.config.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+// const ETHERSCAN_API = process.env.ETHERSCAN_API;
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
```
Comment (nagmak) [inline hardhat.config.js:15]: same for account addresses... is that safe?

**Review by nagmak — APPROVED** [review]
Looks FIREEE. I love the detailed docs. I just had two questions, super minor but otherwise everything is lit 🚀🔥

File: hardhat.config.js (hunk 3 above)
Comment (NickGonzalez04) [reply hardhat.config.js:15]: This one is fine. If you are running the hardhat local node you see the addresses for local dev testing.

File: hardhat.config.js (hunk 2 above)
Comment (NickGonzalez04) [reply hardhat.config.js:6]: Yes and no. Yes, .env is in gitignore. No, bad practice! Thanks for pointing out! Removing and committing changes.

**Pull request merged**